draft-ietf-acme-acme.md

```diff
@@ -2245,6 +2247,39 @@ identifier possession are determined by the server's local policy.
 
 # IANA Considerations
 
+## MIME Type: application/pem-certificate-chain
+
+The "Media Types" registry should be updated with the following additional
+value:
+
+MIME media type name: application
+
+MIME subtype name: pem-certificate-chain
+
+Required parameters: None
+
+Optional parameters: None
+
+Encoding considerations: None
+
+Security considerations: Carries a cryptographic certificate
+
+Interoperability considerations: None
+
+Published specification: draft-ietf-acme-acme
```

Comment:
This should contain a note to the RFC editor to replace the value of the field with `RFC XXXX`, where `XXXX` is the assigned RFC number for this specification.